py/server/deephaven/server/plugin/object/__init__.py

```diff
@@ -39,6 +40,16 @@ def __str__(self):
         return str(self._exporter)
 
 
+class MessageSenderAdapter(MessageSender, ExporterAdapter):
```

Comment:
@jmao-denver This plugin code never went through our review process and is not up to docs specs.

Reply:
Not much doc in this sub-package of deephaven-core. I believe when @devinrsmith wrote it, it was mainly for internal use. Now we are exposing it to the public, it is time to properly document it.  Not sure if this should apply to the dependency https://github.com/deephaven/deephaven-plugin/tree/main/src/deephaven/plugin.

Reply:
Plugin authors are intended to use https://github.com/deephaven/deephaven-plugin/ and not anything from this module

Still probably needs better documentation surrounding it

Reply:
@niloc132 ^^^

Reply:
ty, this is going to get another pass before its ready to go. presently leaning towards doing some api breaking changes, please see https://github.com/deephaven/deephaven-core/pull/4226. I'll be sure to go over these notes before marking it ready for review.